components/todo-item.js

```diff
@@ -1,11 +1,63 @@
-import React from "react";
+import React, { useState } from 'react';
+
 import styled from "styled-components";
 
-export default function TodoItem() {
-  return <Wrapper>Todo</Wrapper>;
+export default function TodoItem({ id, content, date, todoRemove }) {
+  return (
+    <Wrapper>
+      <TodoItemBox>
+        <Todo>{content}</Todo>
+        <Day>{date}
+        <ClearBtn onClick={(() => {todoRemove(id)})}>완료</ClearBtn>
+        </Day>
+      </TodoItemBox>
+    </Wrapper>
+    );
 }
 
 const Wrapper = styled.div`
-  font-size: 18px;
-  border: solid 1px;
+  display: flex;
+`;
+
+const TodoItemBox = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  margin-bottom: 3rem;
+  padding: 1rem 2rem;
+  border-width: 1px;
+  border-style: solid;
+  border-color: black;
+  border-image: initial;
+`;
```

Comment:
```suggestion
const Row = styled.div`
  width: 100%;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  margin-bottom: 3rem;
  padding: 1rem 2rem;
  border-width: 1px solid black;
`;
```
1. ```components/todo-input.js```에 달았던 리뷰와 같은 이유로 네이밍을 ```Row```로 해주시는 게 좋아요.
2. ```align-items:flex-start```는 굳이 안 쓰셔도 style이 똑같을 거예용